br/tests/br_stats/run.sh

```
@@ -51,8 +58,40 @@ load_db1_cnt=$(cat $LOG | grep "restore statistic data done" | grep "${DB}1" | w
-if [ "${load_mark}" -ne "11" ]; then
+if [ "${load_mark}" -ne "22" ]; then
     echo "TEST: [$TEST_NAME] fail on load stats"
     echo $(cat $LOG | grep "restore statistic data done")
     exit 1
 fi
+
+# test load stats is true but statistic data is not backed up
+run_sql "DROP DATABASE ${DB}1;"
+
+rm -f $LOG
+run_br --pd $PD_ADDR restore full -s "local://$TEST_DIR/${DB}2" --log-file $LOG --load-stats=true --filter "${DB}1.br_stats_partition" || cat $LOG
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)